src/xo-app/dashboard/health/index.js

```diff
@@ -450,35 +459,40 @@ export default class Health extends Component {
 
   _getSrUrl = sr => `srs/${sr.id}`
 
-  _getFilteredCollection = createSelector(
-    () => this.state.pools,
-    collections => collections,
-    (pools, collections) =>
-      isEmpty(pools)
-        ? collections
-        : map(collections, collection =>
-          filter(collection, item => includes(resolveIds(pools), item.$pool))
-        )
+  _getPoolFilter = createSelector(
+    createSelector(() => this.state.pools, resolveIds),
+    poolIds =>
+      isEmpty(poolIds)
+        ? identity
+        : collection =>
+          filter(collection, item => includes(poolIds, item.$pool))
+  )
+
+  _getFilteredCollection = collection =>
```

Comment:
They should actually be called `_getFilteredCollectionSelector` and `getCollection` or something like that to match their semantics.